### DISCUSSION TITLE
card individually

### DISCUSSION THREAD

**Discussion by antonioaraujo664** [DeabLabs/cannoli#26] [Q&A]
Is it possible to execute a card individually without executing the others?

**Comment by blindmansion** [comment]
Hey @antonioaraujo664! I don't know if this fits what you were looking for exactly, but if you have a group on the canvas labeled "cannoli", running that canvas will treat the stuff in that group as the only real cannoli nodes. Not sure about all the ins and outs of this feature, it was made as a debugging tool a while back. Hope that's useful!
